code/game/objects/items/stacks/knife.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/stack/throwing_knife
+	name = "throwing knife"
+	gender = NEUTER
```

Comment:
Add a comment explaining why this is here.

Reply:
fucking no idea how to explain this honestly